ui/src/main/webapp/components/CheckboxGroup/CheckboxRow.tsx

```diff
@@ -33,13 +33,13 @@ function CheckboxRow(props: CheckboxRowProps) {
 
     const handleChangeTextBox: TextChangeHandler = (event) => {
         if (event?.target && 'value' in event.target) {
-            handleChange({ field, text: event.target.value, checkbox });
+            handleChange({ field, inputValue: Number(event.target.value), checkbox });
```

Comment:
is there a need to additional check if event.target.value is actually a number, wdyt?
is there a scenario it might be some text?

Reply:
I put a type="number" there and rely on browser number validation. But I found react-ui suggests Number component, so, I will use it instead. Thanks!